### PR TITLE
Various adaptions

### PR DESCRIPTION
**Pull request by FinalAngel**
- use choices as variable name
- concat templates
- fixed renderer on different django versions

### CODE REVIEW

File: djangocms_bootstrap4/contrib/bootstrap4_link/models.py [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
-from .constants import LINK_CHOICES, LINK_SIZES
+from .constants import LINK_CHOICES, LINK_SIZE_CHOICES
 
 
 # 'link' type is added manually as it is only required for this plugin
-COLOR_STYLES = (
+COLOR_STYLE_CHOICES = (
     ('link', _('Link')),
-) + COLOR_STYLES
+) + COLOR_STYLE_CHOICES
```
Comment (vxsx) [inline djangocms_bootstrap4/contrib/bootstrap4_link/models.py:20]: can you really do that? you're effectively overriding an import?

File: private/js/base.js [hunk 2]
```diff
@@ -20,6 +23,35 @@ $(() => {
             'btn-info', 'btn-light', 'btn-dark',
         ],
     });
+    // general color context with aito alignment
```
Comment (vxsx) [inline private/js/base.js]: typo

File: private/js/base.js [hunk 3]
```diff
@@ -7,11 +7,14 @@
 // import 'bootstrap/js/dist/alert'
 import ContextGroup from 'components/context-group';
 import PreviewGenerator from 'components/preview-generator';
-import CardGroup from 'components/card-group'
+import ButtonGroup from 'components/button-group';
 
 
 $(() => {
-    // general color context
+    const static_url = $('.djangocms-bootstrap4').data().static;
```
Comment (vxsx) [inline private/js/base.js]: what's up with the snake case

**Review by vxsx — APPROVED** [review]
in general seems fine

File: private/js/base.js (hunk 3 above)
Comment (vxsx) [reply private/js/base.js]: can be `const { static } = $('...').data();` and then `new Stuff({ static, other: 'options' })`

File: djangocms_bootstrap4/contrib/bootstrap4_link/models.py (hunk 1 above)
Comment (FinalAngel) [reply djangocms_bootstrap4/contrib/bootstrap4_link/models.py:20]: yep you can do that, saw it already on several different addons, just to confirm with @czpython though before merging

**Pull request merged**